src/test/java/org/ossgang/commons/observables/ExceptionHandlersTest.java

```diff
@@ -61,7 +73,7 @@ public void nothingThrows_shouldNotCatchAnything() throws Exception {
     @Test(expected = TimeoutException.class)
     public void mapThrows_subscriberHandles_shouldNotDeflect() throws Exception {
         TestObserver<Integer> testObserver = new TestObserver<>();
-        Property<String> property = Properties.property("A");
+        Property<String> property = Properties.property();
```

Comment:
Out of curiosity - why these changes? I agree the initial value is not needed, but was there anything more behind it?

Reply:
Honestly ...not even sure if I did the change ... I fear that I did some mess during the rebase ... so could it be that this change comes from some of andreas changes ... Sorry.